src/sentry/api/endpoints/project_details.py

```diff
@@ -274,6 +276,31 @@ def put(self, request, project):
                     'sentry:blacklisted_ips',
                     clean_newline_inputs(options['filters:blacklisted_ips'])
                 )
+            if 'filters:releases' in options:
+                if features.has('projects:custom-filters', project, actor=request.user):
```

Comment:
I'd probably use a different switch, yes.

> also, are we planning on having different restrictions on this feature @benvinegar?

I spoke to @dcramer about this. Current plan is this will have the same restrictions as discard. So we need to make sure neither feature goes out to early adopters when it may be locked down to certain accounts afterwards.